Tools/CISupport/buildbot

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/python3
```

Comment:
Adding a wrapper script with exact same name as `buildbot` might be extremely confusing. Better to avoid same name. 

Further, do we need this wrapper script, and change 'buildbot checkconfig' invocations? It might be better to add a build-step to install the required packages (or build-step to invoke a script which does autoinstall) and run any buildbot commands normally after that. That way if anyone try to reproduce any issue manually on the bot, that would also work.

Reply:
This script actually _is_ the `buildbot` script, just with autoinstall hooks.

This is an explicit attempt to avoid installing the required packages because installing the required packages is not intuitive. Getting autoinstall to work with our unit tests means that we're very close to having it work with `buildbot` commands too, we just have to pick a name for the entry script, basically.

Reply:
ok. we should rename it to something explicit for now and use it only in CI. If it works smoothy for a while, then we can consider if we should rename it to buildbot or not.